examples/express/tracer.js

```diff
@@ -45,3 +47,21 @@ module.exports = (serviceName) => {
 
   return opentelemetry.trace.getTracer('express-example');
 };
+
+function filterSampler(filterFn, parent) {
+  return {
+    shouldSample(ctx, tid, spanName, spanKind, attr, links) {
+      if (!filterFn(spanName, spanKind, attr)) {
+        return { decision: opentelemetry.SamplingDecision.NOT_RECORD };
+      }
+      return parent.shouldSample(ctx, tid, name, kind, attr, links);
+    },
+    toString() {
+      return `FilterSampler(${parent.toString()})`;
+    }
+  }
+}
+
+function ignoreHealthCheck(spanName, spanKind, attributes) {
```

Comment:
you could also use https://github.com/open-telemetry/opentelemetry-js/blob/main/packages/opentelemetry-instrumentation-http/src/types.ts#L87 but this solution also works :)